### PR TITLE
Workaround for https://github.com/canonical/dqlite/issues/194

### PR DESCRIPTION
**Pull request by CertainLach**

### CODE REVIEW

**Comment by CertainLach** [comment]
Although this approach is generally bad, I think it is suitable in this case to restore the cluster in case of damage

File: Dockerfile [hunk 1]
```diff
@@ -53,9 +53,8 @@ RUN git clone -b $RAFT_VER https://github.com/canonical/raft.git && \
 
 # --- Build dqlite
 
-RUN git clone --depth 1 https://github.com/canonical/dqlite.git && \
```
Comment (erikwilson) [inline Dockerfile:56]: Why include an unrelated change to the build process? 

This breaks if the DQLITE_VER is a commit hash.

Comment (erikwilson) [reply Dockerfile:56]: Sorry, not important, this is okay for now, with `--depth 1` hashes would not work anyways, we may need to support hashes soon tho.

**Comment by erikwilson** [comment]
Thanks @CertainLach!
